internal/profile/_static/Dockerfile.package-registry

```diff
@@ -1,5 +1,10 @@
 FROM __BASE_IMAGE__
-
 ARG PROFILE
+
+# Disable package validation (already done).
+ENV EPR_DISABLE_PACKAGE_VALIDATION=true
+
 COPY profiles/${PROFILE}/stack/package-registry.config.yml /package-registry/config.yml
 COPY stack/development/ /packages/development
+
+ENTRYPOINT ["./package-registry", "-feature-proxy-mode"]
```

Comment:
It should be also possible to use environment variables to configure this: `EPR_FEATURE_PROXY_MODE=true`. This may be better than setting the entrypoint, as we are already using environment variables to disable validation.

Reply:
Fixed